cast/src/receiver/layout/hc-lovelace.ts

```diff
@@ -94,6 +94,7 @@ class HcLovelace extends LitElement {
     return css`
       :host {
         min-height: 100vh;
+        height: 1px;
```

Comment:
```suggestion
        height: 0;
```

Reply:
Thanks for the review! I just tested `height: 0` and it also seems to work fine.